set.mm

```diff
@@ -409744,6 +409796,35 @@ is a bijection ( 1-1 onto function) in a simple graph.  (Contributed
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
+  $c ComplGraph $.
+  $c ComplUSGraph $.
+
+  $( Extend class notation with (arbitrary) complete graphs. $)
+  ccplgr $a class ComplGraph $.
+
+  $( Extend class notation with complete simple graphs. $)
+  ccusgr $a class ComplUSGraph $.
+
+  ${
+    $d g v $.
+    $( Define the class of all complete "graphs".  A class/graph is called
+       _complete_ if every pair of distinct vertices is connected by an edge,
+       i.e. each vertex has all other vertices as neighbors.  (Contributed by
+       AV, 24-Oct-2020.) $)
+    df-cplgr $a |- ComplGraph = { g | A. v e. ( Vtx ` g )
+                                       v e. ( UnivVtx ` g ) } $.
```

Comment:
I suppose you could also save a bound variable here by writing
```metamath
ComplGraph = { g | ( Vtx ` g ) C_ ( UnivVtx ` g ) }
```
This is slightly cryptic, but in general I think we favor short definition over expressive definitions, and then express the desired property in a more human readable way in a subsequent theorem.

Reply:
Good idea! I will change it in my next PR accordingly.
Since ( UnivVtx ` g ) C_ ( Vtx ` g ) holds anyway, we also can define ComplGraph = { g | ( UnivVtx ` g ) =  ( Vtx ` g ) }.
Which variant would you prefer?

Reply:
I think I would personally prefer the second,
```
ComplGraph = { g | ( UnivVtx ` g ) = ( Vtx ` g ) }
```